app/SigInSignUp/UserActivity.py

```
@@ -1,4 +1,4 @@
-from GoogleAuthRequired import googleAuthRequired
+from app.GoogleAuthRequired import googleAuthRequired
```

Ruff (0.11.9)

`app.GoogleAuthRequired.googleAuthRequired` imported but unused

Remove unused import: `app.GoogleAuthRequired.googleAuthRequired`

(F401)